### PR TITLE
Mcp 2115 - Clean up sec-lighthouse-api integrationTest.

### PR DESCRIPTION
**Pull request by wlin2019**
IntegrationTest properties clean up

## What was the problem?
See comments in PR #817.
TODO: run the tests with `@ActiveProfiles("integrationTest")` and remove unnecessary settings below
#   See application-test.properties for an example.
#   May even be able to use `@ActiveProfiles({"test","integrationTest"})` to remove more settings

Associated tickets or Slack threads:
MCP-2115

## How does this fix it?[^1]
Removed the unnecessary setting in the property file for integration test, and kept lh related as the credential values used in test are faked and cannot be used to connect Lighthouse FHIR API sandbox used in the integration test.

## How to test this PR
- Step 1. build app and launch it. The app should be up in local dev environment.
- Step 2. run ./gradlew integrationTest. The test should be successful. Jacco coverage report should show a decent coverage.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: svc-lighthouse-api/src/integrationTest/java/gov/va/vro/abddataaccess/FhirClientTest.java [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package gov.va.vro.abddataaccess;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (yoomlam) [inline svc-lighthouse-api/src/integrationTest/java/gov/va/vro/abddataaccess/FhirClientTest.java:3]: Undo this change due to [warning](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/3721229093/jobs/6311241939#step:5:880)

**Comment by yoomlam** [comment]
@wlin2019 I'm getting assertion errors -- off by 1 hour:
```
AbdApplicationIt > testForClaim01() FAILED
    java.lang.AssertionError: medications[0].authoredOn
    Expected: 1950-04-05T23:00:00Z
         got: 1950-04-05T22:00:00Z
     ; medications[1].authoredOn
    Expected: 1950-04-06T07:24:55Z
         got: 1950-04-06T06:24:55Z
     ; medications[2].authoredOn
    Expected: 1952-04-11T23:00:00Z
         got: 1952-04-11T22:00:00Z
     ; medications[3].authoredOn
    Expected: 1952-04-11T23:00:00Z
         got: 1952-04-11T22:00:00Z
     ; medications[4].authoredOn
    Expected: 1952-04-12T07:24:55Z
         got: 1952-04-12T06:24:55Z
     ; medications[5].authoredOn
    Expected: 1952-04-12T07:24:55Z
         got: 1952-04-12T06:24:55Z
     ; medications[6].authoredOn
    Expected: 1953-04-25T23:00:00Z
         got: 1953-04-25T22:00:00Z
     ; medications[7].authoredOn
    Expected: 1953-04-26T08:24:55Z
         got: 1953-04-26T07:24:55Z
     ; medications[8].authoredOn
    Expected: 2009-08-04T00:00:00Z
         got: 2009-08-03T23:00:00Z
     ; medications[9].authoredOn
    Expected: 2009-08-04T08:24:55Z
         got: 2009-08-04T07:24:55Z
     ; medications[10].authoredOn
    Expected: 2012-11-16T00:00:00Z
         got: 2012-11-15T23:00:00Z
     ; medications[11].authoredOn
    Expected: 2012-11-16T08:24:55Z
         got: 2012-11-16T07:24:55Z
     ; medications[12].authoredOn
    Expected: 2014-11-27T00:00:00Z
         got: 2014-11-26T23:00:00Z
     ; medications[13].authoredOn
    Expected: 2014-11-27T08:24:55Z
         got: 2014-11-27T07:24:55Z
        at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:417)
        at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:394)
        at gov.va.vro.abddataaccess.AbdApplicationIt.testForClaim01(AbdApplicationIt.java:111)
```

File: svc-lighthouse-api/src/integrationTest/java/gov/va/vro/abddataaccess/FhirClientTest.java (hunk 1 above)
Comment (wlin2019) [reply svc-lighthouse-api/src/integrationTest/java/gov/va/vro/abddataaccess/FhirClientTest.java]: done.

**Comment by yoomlam** [comment]
@wlin2019 Does the integration test succeed for you?

**Comment by wlin2019** [comment]
> @wlin2019 Does the integration test succeed for you?

It did succeed yesterday. I pulled the code from the branch a few minutes ago and ran 
   ./gradlew clean integrationTest. 
It was successful. 

I've never seen the testForClaim01() assertion failed in integrationTest in my local. 

But It seem that some tests in integrationTest fail in Intellij (right click on the test and choose run or debug) as it seems not smart enough to pick up LH setting after they were removed from application-integrationTest.properties. But they may not be important at this point.

**Comment by yoomlam** [comment]
I start VRO locally (with all 8 containers): `./gradlew app:dcPrune app:dockerComposeUp`
Then when I run: `./gradlew :svc-lighthouse-api:integrationTest --rerun-tasks -x test --info`
it fails:
```
AbdApplicationIt > testForClaim01() FAILED
    java.lang.AssertionError: medications[0].authoredOn
    Expected: 1950-04-05T23:00:00Z
         got: 1950-04-05T22:00:00Z
     ; medications[1].authoredOn
...
    Expected: 2012-11-16T08:24:55Z
         got: 2012-11-16T07:24:55Z
     ; medications[12].authoredOn
    Expected: 2014-11-27T00:00:00Z
         got: 2014-11-26T23:00:00Z
     ; medications[13].authoredOn
    Expected: 2014-11-27T08:24:55Z
         got: 2014-11-27T07:24:55Z
        at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:417)
        at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:394)
        at gov.va.vro.abddataaccess.AbdApplicationIt.testForClaim01(AbdApplicationIt.java:111)

...
3 tests completed, 1 failed
...
* What went wrong:
Execution failed for task ':svc-lighthouse-api:integrationTest'.

```
